pkg/util/log/log.go

```diff
@@ -97,7 +97,10 @@ func CheckFatal(location string, err error, logger log.Logger) {
 		logger = log.With(logger, "msg", "error "+location)
 	}
 	// %+v gets the stack trace from errors using github.com/pkg/errors
-	logger.Log("err", fmt.Sprintf("%+v", err))
+	errStr := fmt.Sprintf("%+v", err)
+	fmt.Fprintln(os.Stderr, errStr)
```

Comment:
Q: wondering putting same error in log and in stderr can cause scraping it twice? based on container runtime?

My understanding is (for example in OCI) both stderr and stdout of an application goes into pod logs which will be scraped by promtail.

Reply:
What is your concern, specifically?

The point here is that when we log the error using our standard logger, it replaces all newlines and tabs with `\n` and `\t` respectively, so that it conforms with the `logfmt` format.

Reply:
Yea I understand that from the change. :+1: 

My concern is before this change, ony `logfmt` format of that error will be scraped and send to Loki, but now both "logfmt" and "plain" format of the error will be send to Loki.

Reply:
Sure, but what is your actual concern with that? That seems like a perfectly reasonable tradeoff for improved readability.

Reply:
Not a big concern. Just wanted to clarify. that's all.